nsqadmin/static/html/index.html

```diff
@@ -13,7 +14,7 @@
     <div id="container"></div>
 
     <script type="text/javascript">
-        var USER_AGENT = 'nsqadmin';
+        var USER_AGENT = 'nsqadmin v{{.Version}}';
```

Comment:
`nsqadmin/{{.Version}}` to follow [rfc7231 section 5.5.3](http://tools.ietf.org/html/rfc7231#section-5.5.3)